src/inspector.rs

```diff
@@ -598,6 +598,17 @@ impl Debug for V8InspectorClientBase {
 pub struct V8InspectorSession(Opaque);
 
 impl V8InspectorSession {
+  // Taken directly from https://source.chromium.org/chromium/chromium/src/+/main:v8/src/inspector/v8-inspector-session-impl.cc;l=66;drc=87132919a42f0ddedee557145ee3d8336a8320c7
+  pub fn can_dispatch_method(&self, method: StringView) -> bool {
```

Comment:
```suggestion
  pub fn can_dispatch_method(method: StringView) -> bool {
```

It is a static in V8 AFAIK.